documentation/src/main/wiki/Splitter.md

```diff
@@ -4,14 +4,14 @@ goes to Knot context and can be processed later by [[Knots|Knot]].
 
 ##How does it work?
 HTML Fragment Splitter gets a Knot context as input and responds with the modified Knot context. It 
```

Comment:
`HTML Fragment Splitter gets a Knot context as input and responds with the modified Knot context.`
to
`HTML Fragment Splitter reads Knot Context having raw HTML, splits it into fragments, updates Knot Context, and returns back to the caller.`